src/runtime/composables/usePrecognitionForm.ts

```diff
@@ -145,7 +144,7 @@ export const usePrecognitionForm = <T extends Payload>(
       return toRaw(form.fields) as T
     },
 
-    setData(data: PayloadData<T>): PrecognitionForm<T> {
+    setData(data: Partial<T>): PrecognitionForm<T> {
```

Comment:
I believe we should use `Partial<PayloadData<T>>` here for better consistency

Reply:
hmm, I should have thought of that 🤔 done!